scripts/normalize-links.sh

```diff
@@ -28,23 +28,25 @@ fi
 # - href=
 # followed by an optional double quote
 
+# We had to add the js checks for 17.09 and higher when we use JS for the TOC
+
 printf "Cleaning up $VER"
 
 # Fix relative links for archive
-printf "."; find ${TARGET} -type f -name '*.html' -print0 | xargs -0 sed -i 's#\(src\|href\)=\("\{0,1\}\)\(http\|https\)://\(docs\|docs-stage\).docker.com/#\1=\2/#g'
+printf "."; find ${TARGET} -type f \( -name '*.html' -or -name '*.js' \) -print0 | xargs -0 sed -i 's#\(src\|href\)=\("\{0,1\}\)\(http\|https\)://\(docs\|docs-stage\).docker.com/#\1=\2/#g'
```

Comment:
Hm don't think, this will work, because the paths are in JSON, and not `src=` / `href=`;

```
"path":"/engine/tutorials/networkingcontainers/"
```

Reply:
Ahh, good catch